mailpoet/tests/integration/Tasks/Subscribers/BatchIteratorTest.php

```diff
@@ -46,14 +57,6 @@ public function testItIterates() {
     foreach ($this->iterator as $batch) {
       $i++;
 
-      // process subscribers
```

Comment:
I wonder if it's possible to remove this without replacement. I'm curious only.

Reply:
I was trying to understand why we would need to change the value of the field `processed` for the subscribers inside this test. I couldn't figure this out, but in the process, I realized that the test didn't require it to pass (even before the changes that I made). So that is why it seems to me it is safe to remove this part of the code.